packages/get-uri/package.json

```diff
@@ -51,8 +51,7 @@
   "dependencies": {
     "basic-ftp": "^5.0.2",
     "data-uri-to-buffer": "^6.0.2",
-    "debug": "^4.3.4",
-    "fs-extra": "^11.2.0"
```

Comment:
`@types/fs-extra` can also be removed from `devDependencies`.

Reply:
Good spot, thanks!